packages/design-system/src/css/checkbox.scss

```diff
@@ -190,6 +190,7 @@
 		position: relative;
 		border: var.$checkbox-input-border;
 		border-radius: var.$checkbox-border-radius;
+		margin-top: 2px;
```

Comment:
nit: `--spacing-5xs` 😛

Reply:
Missed this one! Can you reapprove? Thank you!